feincms3_sites/models.py

```diff
@@ -150,3 +154,13 @@ def save(self, *args, **kwargs):
         super().save(*args, **kwargs)
 
     save.alters_data = True
+
+    @staticmethod
+    def add_site_field(sender, **kwargs):
+        model = settings.FEINCMS3_SITES_SITE_MODEL
+        if (sender._meta.app_label + '.' + sender._meta.object_name) == model:
```

Comment:
`sender._meta.label_lower == model.lower()` or something like that? I think you shouldn't depend on the exact capitalization of model names.

Reply:
Yes!

Reply:
done